composer.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Does this file have to be in the root? We'd like to keep it as uncluttered as possible.